doc/source/cluster/kubernetes/examples/tpu-serve-stable-diffusion.md

```diff
@@ -0,0 +1,77 @@
+(kuberay-tpu-stable-diffusion-example)=
+
+# Serve a Stable Diffusion model on GKE with TPUs
+
+> **Note:** The Python files for the Ray Serve app and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples). This example is adapted from the [tensorflow/tpu](https://github.com/tensorflow/tpu/tree/master/tools/ray_tpu/src/serve) Cloud TPU example.
```

Comment:
```suggestion
> **Note:** The Python files for the Ray Serve app and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples). This example adapts the [tensorflow/tpu](https://github.com/tensorflow/tpu/tree/master/tools/ray_tpu/src/serve) to use cloud TPUs.
```

Reply:
Is this rewording still technically accurate?

Reply:
Both examples (this one and the tensorflow/tpu example ) use Cloud TPUs, I can reword it to be similar to the suggestion though.

Reply:
[e2fe072](https://github.com/ray-project/ray/pull/45924/commits/e2fe0720674aca7f7ce0ba09c0eaeb5a510c7168)